### ISSUE TITLE
Unexpected list of results with Search box on home page

### ISSUE THREAD

**Issue by saubin78** [agroportal/project-management#594]
**Describe the bug**
If you type "phenological scales of olive tree" then press enter in the search box which is on the home page, the results do not include the ontologies that match this string.  

**To Reproduce**
1. Go to Home
2. Type "phenological scales of olive tree" then Click on 'Enter'
3. look at results page
4. See that only classes and concepts are listed. Ontologies are missing. 

**Expected behavior**
I'd expect to get all the ontologies that have "phenological scales of olive tree" in their name, i.e. BBCH phenological scales of olive tree which appears in the autocompletion results on home page. I'd expect the same behavior from autocompletion and "enter" action

**Screenshots**
On home page - autocompletion --> ontologies
![image](https://github.com/user-attachments/assets/d480a5b1-6f96-42c4-b685-86ccafa26d0a)
On results page - after pressing "enter" --> no ontologies
![image](https://github.com/user-attachments/assets/76425811-8204-4049-bad4-f22dfe0f1f9c)

**Comment by jonquet** [comment]
This is not a bug. If you end up on the Search page it means you have (in the home page "search box" ) clicked "Search ontology content" in the proposed menu. 
So you endup in the ontology search page which search in ontologies and does not search for ontologie (it never did).

To get alll the ontologies that have "phenological scales of olive tree" in their name (with such a long name you already see the result in the menu => PPD-OLT) you would have to select "Browse all ontologies" in the proposed menu.

**Comment by jonquet** [comment]
I am concerned experimented users don't necessrily get it ;)
Is this search box to advance?

**Comment by saubin78** [comment]
I understand that this is not a bug nor a too advanced search box. The problem is that the behavior is implicit to the user in the sense that I have not clicked "Search ontology content" in the proposed menu. 
1) The search box says : "Search an ontology or a term". 
2) I've copy-pasted the name of the ontology I was looking for and **pressed "Enter"** 
3) I expected to have either ontologies or terms as it is mentioned in the search box : "Search an ontology or a term". 
4) I got only terms as results

**Comment by jonquet** [comment]
Indeed, you you press enter after filling the menu the default behaviour is to bring you to the /search page which is indeed the search service for the ontology content. 
We do not have a page or service to search in AgroPortal across objects. Even the /search page does not even returns other things than classes/concepts within an ontology (and not instance, properties, schemes and collections). 
Going toward such a page or search service may be desired and is discussed sometime. But its not trivial and its a big change.

**Comment by jonquet** [comment]
- [ ] Un mini detail qu'on peut faire la est de surligner par default la ligne "Search ontology content" car c'est elle qui sera activé quand on cliquera entrer et changer ce surlignage quand on commence a decendre dans le menu

- [x] Changer la phrase de doc qur la page de Serach pour bien dire explicitement quo'n a cherché DANS des ontologies et pas pour des ontologies 
<img width="492" alt="Capture d’écran 2024-08-02 à 16 38 43" src="https://github.com/user-attachments/assets/74553074-1f0d-42d4-a74c-763f45cce74a">

**Comment by saubin78** [comment]
Hi. Again, I ended in an unexpected (from my point of you) and inconsistent behavior of the home page search box. 
When I enter "ATOL" (looking for the ATOL ontology by its acronym) on the first time I can see the ATOL ontology in the result for a quarter of seconds then it disappears. 
![image](https://github.com/user-attachments/assets/a0abcc65-22a5-4e89-9e07-c1c40543ffa7)
Then I try a second time and I can see it
![image](https://github.com/user-attachments/assets/7e75bb1e-0b57-42e1-9fc9-7b7219e597ed)

**Comment by jonquet** [comment]
Oui. J'ai observé ce comportement aussi. Un résultat pertinent apparait tout de suite puis est remplacé par d'autres. Et comme il y en a que 3, il ne reste pas.

**Comment by syphax-bouazzouni** [comment]
Duplicate of https://github.com/agroportal/project-management/issues/603

**Comment by jonquet** [comment]
Avant que l'on archive cette issue (qui contenait plusieurs chose) on devrait essayer d'améliorer le comportement par défaut de la search box. 
Ce qui surprend @saubin78 ici est que le fait qu'on tape "entrée" dans le menu nous emmène par défaut a la page  /search. 
Aussi, si je tape: "Plant Ontology + Entrée" je finis sur la page search avec des resultats qui correpsondent au contenu des ontologies pas à la liste des ontologies. 

Ma proposition serait qu'on change le comportement par défaut de "Entrée" pour que cela ouvre le PREMIER resultat du popup. 
Ainsi, si je tape "Plant Ontology + Entrée"
